samples/bluetooth/advertiser/boards/rv32m1_vega_ri5cy.overlay

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
just being curious, no changes requested.

I see several bluetooth samples, but not all, having this dts overlay file.

Do you know when this file is required ?
Or are you just including this file because it was present in the sample you used as template.

Reply:
Yeah, I use nRF5340 DK as my target. Even I'm not sure when this overlay would be used.